docs/ocid.8.md

```diff
@@ -54,7 +54,7 @@ ocid is meant to provide an integration path between OCI conformant runtimes and
   path to configuration file
 
 **--conmon**=""
-  path to the conmon executable (default: "/usr/libexec/ocid/conmon")
+  path to the conmon executable (default: "/usr/local/libexec/ocid/conmon")
```

Comment:
I hate  to change this, since the installed version via RPM will put them in /usr/libexec/ocid/conmon.  Perhaps we should just remove the default: